backend/src/Designer/Controllers/PolicyController.cs

```diff
@@ -0,0 +1,170 @@
+﻿using Altinn.Authorization.ABAC.Xacml;
+using Altinn.Studio.Designer.Services.Interfaces;
+using Altinn.Studio.PolicyAdmin;
+using Altinn.Studio.PolicyAdmin.Models;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Altinn.Studio.Designer.Controllers
+{
+    [Route("api/[controller]")]
```

Comment:
When splitting the endpoints so separate org/app vs resource paths, would it not make more sense to also move the controller level route to each method?

Reply:
Removed route completly